appveyor.yml

```diff
@@ -0,0 +1,208 @@
+# As config was originally based on an example by Olivier Grisel. Thanks!
+# https://github.com/ogrisel/python-appveyor-demo/blob/master/appveyor.yml
+
+clone_depth: 50
+
+# No reason for us to restrict the number concurrent jobs
+max_jobs: 100
+
+environment:
+  global:
+      MINGW_32: C:\mingw-w64\i686-6.3.0-posix-dwarf-rt_v5-rev1\mingw32\bin
+      MINGW_64: C:\mingw-w64\x86_64-6.3.0-posix-seh-rt_v5-rev1\mingw64\bin
+      OPENBLAS_32: https://github.com/xoviat/lapack/releases/download/latest/Lapack.zip
```

Comment:
Copied to https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/Lapack.zip - is that what you mean?